lib/txmgr/utils.go

```diff
@@ -0,0 +1,63 @@
+package txmgr
+
+import (
+	"sort"
+
+	"github.com/omni-network/omni/contracts/bindings"
+	"github.com/omni-network/omni/lib/xchain"
+)
+
+// SubmissionToBinding converts a go xchain submission to a solidity binding submission.
+func SubmissionToBinding(sub xchain.Submission) bindings.XTypesSubmission {
```

Comment:
only relayer will use this, so it should live in relayer.

Reply:
This + not a fan of "utils" as it becomes a dumpster for code